app/src/main/java/com/framgia/moviedb/screen/home/MovieAdapter.java

```diff
@@ -52,28 +55,43 @@ public int getItemCount() {
         return mMovies != null ? mMovies.size() : 0;
     }
 
-    static class ViewHolder extends RecyclerView.ViewHolder {
+    static class ViewHolder extends RecyclerView.ViewHolder implements View.OnClickListener {
         private ImageView mImageView;
         private Context mContext;
+        private OnItemClick mOnItemClick;
+        private Movie mMovie;
 
-        public ViewHolder(Context context, @NonNull View itemView) {
+        public ViewHolder(Context context, @NonNull View itemView, OnItemClick onItemClick) {
             super(itemView);
             mContext = context;
+            mOnItemClick = onItemClick;
             mImageView = itemView.findViewById(R.id.image_movie);
         }
 
+
+        @Override
+        public void onClick(View v) {
+            mOnItemClick.onItemClick(mMovie);
+        }
+
         public void bindData(Movie movie) {
             if (movie == null) {
                 return;
             }
+            mImageView.setOnClickListener(this);
+            mMovie = movie;
             String url = APIUtils.PRE_POSTER_URL + movie.getPosterPath();
             GlideApp.with(mContext)
                     .load(url)
                     .apply(new RequestOptions()
                             .placeholder(R.drawable.ic_loading_icon)
                             .error(R.drawable.ic_error_icon)
-                            .override(700, 300))
+                            .override(SIZE_IMAGE))
                     .into(mImageView);
         }
     }
+
+    interface OnItemClick {
```

Comment:
```suggestion
    interface OnItemClickListener {
```

callback nên đặt là listener
giống như OnClickListener, OnLongClickLístener, OnTouchListẻner